### PR TITLE
[SE-0304] Suggest changes to 1st-half of proposal

### PR DESCRIPTION
**Pull request by benrimmington**

### CODE REVIEW

File: proposals/0304-structured-concurrency.md [hunk 1]
```diff
@@ -164,19 +164,19 @@ Of course, a function’s task may itself be a child of another task, and its pa
 
 In this proposal, the way to create child tasks is only within a `TaskGroup`, however there will be a follow-up proposal that enables creation of child tasks in any asynchronous context.
 
-### Partial tasks
+### Jobs
```
Comment (ktoso) [inline proposals/0304-structured-concurrency.md:167]: Not sure if we actually want this rename.

I'll double check.

Comment (benrimmington) [reply proposals/0304-structured-concurrency.md:167]: > Not sure if we actually want this rename.

@ktoso https://github.com/apple/swift-evolution/pull/1368#discussion_r637101407

Comment (ktoso) [reply proposals/0304-structured-concurrency.md:167]: Ah ok, great then :)

**Review by ktoso — APPROVED** [review]
Thanks, that's all correct 👍

**Comment by ktoso** [comment]
Heh this is to the PR I see...

Could you make the same PR to the main branch of evolution @benrimmington ? 
The team had merged the proposal already.

**Comment by ktoso** [comment]
Actually, let me port that 👍

**Pull request merged**